src/LibraryManager.Vsix/Microsoft.Web.LibraryManager.Vsix.csproj

```diff
@@ -184,7 +184,7 @@
     <PackageReference Include="Microsoft.WebTools.Languages.Json.VS" />
     <PackageReference Include="Newtonsoft.Json" />
     <PackageReference Include="Nuget.VisualStudio" />
-  </ItemGroup>
+   </ItemGroup>
```

Comment:
nit: unnecessary change in this file (and bad indent)

Reply:
Correct and corrected.